.github/workflows/pr_ruby.yml

```
@@ -0,0 +1,37 @@
+name: Ruby PR Check
+
+on:
+  push:
+    branches: [ master ]
+    paths: ["ruby/**"]
+  pull_request:
+    branches: [ master ]
+    paths: ["ruby/**"]
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./ruby
+    timeout-minutes: 5
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@359bebbc29cbe6c87da6bc9ea3bc930432750108
+        with:
+          ruby-version: '3.1'
+      - name: Install dependencies
+        run: bundle install
+        working-directory: ${{ env.working-directory }}
+      - name: Test
```

Ini direname aja soalnya ternyata step ini juga report ke codecov
```suggestion
      - name: Test and report coverage
```